CHANGES.txt

```diff
@@ -1,4 +1,4 @@
-﻿Current
```

Comment:
Please, keep this line.
And you can add your name to the list. You made the job, you deserve it! :)